Composer/ScriptHandler.php

```diff
@@ -254,33 +254,18 @@ public static function doBuildBootstrap($bootstrapDir)
             'Symfony\\Component\\HttpFoundation\\FileBag',
             'Symfony\\Component\\HttpFoundation\\ServerBag',
             'Symfony\\Component\\HttpFoundation\\Request',
-            'Symfony\\Component\\HttpFoundation\\Response',
-            'Symfony\\Component\\HttpFoundation\\ResponseHeaderBag',
 
-            'Symfony\\Component\\DependencyInjection\\ContainerAwareInterface',
-            'Symfony\\Component\\DependencyInjection\\Container',
             'Symfony\\Component\\HttpKernel\\Kernel',
             'Symfony\\Component\\ClassLoader\\ClassCollectionLoader',
             'Symfony\\Component\\ClassLoader\\ApcClassLoader',
-            'Symfony\\Component\\HttpKernel\\Bundle\\Bundle',
```

Comment:
but this is too late, because the kernel loads the bundle to be able to build the list of compiled classes (as it asks bundles for them), so the class is already used before that